src/DataTool.php

```diff
@@ -131,24 +132,10 @@ public function generateData()
             if (!empty($data['source']) && $data['source'] != 'custom_fields') {
                 continue;
             }
-
-            $type = (!empty($data['dbType'])) ? $data['dbType'] : $data['type'];
-            $GLOBALS['fieldData'] = $data;
-
-            /* There are 3 unique parts to the seed: the Module name,
-             * the count of the record, and the name of the field.
-             * Using these 3 things should keep our seed unique enough.
-             */
-            $seed = $this->generateSeed($this->module, $field, $this->count);
-            $value = $this->getData($field, $type, $data['type'], $seed);
-            if (!empty($value) || $value == '0') {
-                if (empty($data['source'])) {
-                    $this->installData[$field] = $value;
-                } else {
-                    // "source" == "custom_fields" is an indicator for Custom field in VarDefs
-                    $this->installDataCstm[$field] = $value;
-                }
+            if (!empty($this->installData[$field])) {
```

Comment:
`$this->installData[$field]` can also contain `"0"`. In this case `empty("0")` returns true and the field value is generated again.

Let's change this check to use `isset` rather than `empty`: `if (isset($this->installData[$field])) {...`